gradle/libs.versions.toml

```diff
@@ -1,4 +1,7 @@
 [versions]
+robolectric-compat = "4.10"
```

Comment:
I remember it has been upgraded to 4.10.2.

Reply:
Fixed at 8e88305c